### DISCUSSION TITLE
CSIRO grassfire experiment particle placement

### DISCUSSION THREAD

**Discussion by bjmvandongen** [firemodels/fds#13327] [Q&A]
Hi all, 

I am simulating the F19 case of the CSIRO grassfire experiments. 
The fuel bed height is 0.51 meters and the grid size that I use is 1 m x 1m x 1m.  When I specify 1 particle per cell and also cell-centered, where is the particle placed? If the particle is placed in the center of the cell, is the fuel bed height then modelled as 1 meter or still as 0.51 meter? It does not really matter if I specify cell-centered right? because then the particle is placed randomly within the cell. 
Hopefully someone can give me some clarification.

**Comment by drjfloyd** [comment]
The particle location in the cell only has an impact if the particle is moving . The particle only sees the cell mean quantities (temp, species, gas velocity, etc.) there is no interpolation of those for the actual particle location.

**Comment by ericvmueller** [comment]
There is a distance weighted average taken for the gas velocity from the
nearest relevant faces/vectors in part.f90. So in that sense location would
matter for drag. I don’t believe we do the same for convection though.

**Reply by bjmvandongen** [reply]
Thank you for the reply. 
So the particle placement is important for the drag, but a 1 meter high cell can still be used to simulate 0.51 meter high vegetation?

**Reply by bjmvandongen** [reply]
or because the cell is 1 meter high, the vegetation is modelled as 1 meter high?

**Reply by bjmvandongen** [reply]
&INIT PART_ID='blade of grass', XB=0.,200.,-100.,100.,0.00,0.51, N_PARTICLES_PER_CELL=1, CELL_CENTERED=T, MASS_PER_VOLUME=0.6137, DRY=T /

this is how I specified it

**Reply by ericvmueller** [reply]
FDS will use the 'XB' and 'MASS_PER_VOLUME' you request to determine the total mass. But there is no special treatment of partially filled grid cells, so the mass is then redistributed in all the cells that intersect the 'XB'. So you should end up with a taller layer of grass but a lower bulk density, to conserve mass. You should be able to confirm this by outputting 'PARTICLE BULK DENSITY'.
